getting-started.adoc

```diff
@@ -1,154 +1,192 @@
 [[getting-started-with-raincatcher]]
-Getting Started with Raincatcher
---------------------------------
+= Getting Started with Raincatcher
 
-[[overview]]
-Overview
-~~~~~~~~
+== Overview
+
+Raincatcher is a set of templates and a field workforce management (WFM) application that connects a business' back-office with its fleet of mobilized employees.  Raincatcher leverages mobile hardware, making use of each employee's mobile phone.
+
+Raincatcher consists of a set of reusable modules that can be included in your application using http://browserify.org/[browserify]. 
+
+
+Raincatcher is based on the principle of multiple,
+loosely-coupled modules communicating through the mediator pattern.
+Raincatcher modules make use of the
+https://addyosmani.com/largescalejavascript/[mediator pattern] to enable
+loose coupling between the Raincatcher modules and their consuming
+applications / modules as shown in the following diagram:
+
+image:assets/images/wfm-architecture.png[Raincatcher Architecture]
 
-Raincatcher is a field workforce management (WFM) application that connects a business' back-office with its fleet of mobilized employees.  Raincatcher leverages commoditized mobile hardware, making use of each employee's mobile phone.
 
 [[raincatcher-structure]]
-Raincatcher Structure
-~~~~~~~~~~~~~~~~~~~~~
+== Raincatcher Structure
 
-Raincatcher consists of a set of reusable modules that are designed to be included in your application using [browserify](http://browserify.org/). Raincatcher modules are packaged and distributed via [npm](https://www.npmjs.com/) and export one or more of:
+Raincatcher modules are packaged and distributed via [npm](https://www.npmjs.com/) and export:
```

Comment:
This link is not rendering properly:

<img width="849" alt="screen shot 2017-03-01 at 3 56 15 pm" src="https://cloud.githubusercontent.com/assets/586800/23478447/cd63cc1a-fe97-11e6-94bb-3f127acc3317.png">